### ISSUE TITLE
differentiable type casting

### ISSUE THREAD

**Issue by kausic94** [tensorflow/tensorflow#23274]
There seems to be no way to do differentiable type casting in tensorflow. tf.cast is not differentiable and so is tf.image.convert_image_dtype() to cast image tensors. How to incorporate casting in a loss function if there are no options ?

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
